### ISSUE TITLE
Cannot read properties of undefined

### ISSUE THREAD

**Issue by Kayxue** [facundoolano/google-play-scraper#493]
* Operating System: Windows 10 Build 18363.1379
* Node version:17.1.0
* google-play-scraper version: 8.0.4

### Description: 
Cannot read properties of undefined

### Example code (TypeScript):
```ts
import googlePlay from "google-play-scraper"
googlePlay.app({
    appId:"com.soulgamechst.majsoul",
    country: "tw",
    lang: "zh-tw"
}).then(console.log)
```

### Error message:

```
C:\Users\user\Desktop\Project\node_modules\google-play-scraper\lib\mappers\details.js:125
  const html = cheerio.load('<div>' + description.replace(/<br>/g, '\r\n') + '</div>');
                                                  ^
TypeError: Cannot read properties of undefined (reading 'replace')
    at Object.descriptionText [as fun] (C:\Users\user\Desktop\Project\node_modules\google-play-scraper\lib\mappers\details.js:125:51)
    at C:\Users\user\Desktop\Project\node_modules\google-play-scraper\lib\utils\scriptData.js:52:19
    at XWrap.f (C:\Users\user\Desktop\Project\node_modules\ramda\dist\ramda.js:5955:28)
    at XWrap.@@transducer/step (C:\Users\user\Desktop\Project\node_modules\ramda\dist\ramda.js:392:25)
    at _arrayReduce (C:\Users\user\Desktop\Project\node_modules\ramda\dist\ramda.js:5137:46)
    at _reduce (C:\Users\user\Desktop\Project\node_modules\ramda\dist\ramda.js:5167:24)
    at map (C:\Users\user\Desktop\Project\node_modules\ramda\dist\ramda.js:5954:20)
    at C:\Users\user\Desktop\Project\node_modules\ramda\dist\ramda.js:586:23
    at Object.f2 [as map] (C:\Users\user\Desktop\Project\node_modules\ramda\dist\ramda.js:473:22)
    at extractFields (C:\Users\user\Desktop\Project\node_modules\google-play-scraper\lib\utils\scriptData.js:40:14)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by qw4n7y** [comment]
+1

**Comment by icarcal** [comment]
This is also fixed by the PR #498 .
It's already merged.
Needs publish on npms

**Comment by UnknownKID69** [comment]
same
